cdc/redo/writer/writer.go

```diff
@@ -93,7 +93,9 @@ type LogWriter struct {
 	cfg       *LogWriterConfig
 	rowWriter fileWriter
 	ddlWriter fileWriter
-	storage   storage.ExternalStorage
+	// storage in LogWriter is used to clean up the
```

Comment:
Maybe this comment is not accurate. Because use also use `storage` to save metadata. See function `writeMetaToS3()`